UPGRADING.md

```diff
@@ -266,6 +267,26 @@ The module name is assumed by `baseapp` to be the second element of the message
 In case a module does not follow the standard message path, (e.g. IBC), it is advised to keep emitting the module name event.
 `Baseapp` only emits that event if the module have not already done so.
 
+### `x/params`
+
+The `params` module was deprecated since v0.46. The Cosmos SDK has migrated away from `x/params` for its own modules.
+The `params` module will be removed in `v0.48`, as mentioned [in v0.46 release](https://github.com/cosmos/cosmos-sdk/blob/v0.46.1/UPGRADING.md#xparams).
+
+Cosmos SDK modules now store their parameters directly in its repective modules. In order to migrate these modules, you need to add the following when instantiating the params module in your `app.go`:
+
+```go
```

Comment:
Maybe we could improve https://github.com/cosmos/cosmos-sdk/blob/main/simapp/upgrades.go instead?
Trying something.

Reply:
Done in [5dff820](https://github.com/cosmos/cosmos-sdk/pull/14992/commits/5dff82044d5759c5df845b08b227721767bdddee), wdyt?

Reply:
@julienrbrt LGTM!

Reply:
Let's change